lib/chat/index.js

```diff
@@ -157,24 +159,21 @@ class ChatBot extends BaseBot {
    */
   async watchAllChannelsForNewMessages(onMessage: OnMessage) {
     await this._guardInitialized()
-    keybaseExec(this.homeDir, ['chat', 'api-listen'], {
-      onStdOut: (line: string) => {
-        try {
-          const messageObject = JSON.parse(line)
-          if (this.username && messageObject.msg.sender.username !== this.username.toLowerCase()) {
-            onMessage(messageObject.msg)
-          }
-        } catch (error) {
-          console.error(error)
+    const child = spawn('keybase', ['--home', this.homeDir, 'chat', 'api-listen'])
```

Comment:
I think it makes sense to not use `keybaseExec` for the message watching methods. Unlike most other keybase calls, we don't really care about the return type, we just want our callback to fire on every new line.

Not only does this give us the advantage of being able to add the children easily to our `spawnedProcesses` array, but I think this will make error handling easier for the watch methods, as we don't have to worry about the service erroring on deinit.